ffi/examples/read-table/schema.h

```diff
@@ -148,7 +148,8 @@ void visit_array(
 {
   SchemaBuilder* builder = data;
   char* name_ptr = malloc(sizeof(char) * (name.len + 22));
-  snprintf(name_ptr, name.len + 1, "%s", name.ptr);
+  int name_chars = name.len > INT_MAX ? INT_MAX : (int)name.len; // handle _REALLY_ long names by truncation
```

Comment:
```suggestion
  // NOTE: we truncate to the max int size because the format specifier "%.*s" requires an int length specifier
  int name_chars = name.len > INT_MAX ? INT_MAX : (int)name.len; // handle _REALLY_ long names by truncation
```